package.json

```diff
@@ -19,8 +19,8 @@
     "chai": "~3.5.0",
     "chai-as-promised": "~5.3.0",
     "chai-things": "~0.2.0",
-    "mocha": "^6.2.0",
-    "nock": "^10.0.6"
+    "nock": "^10.0.6",
+    "mocha": "^8.0.1"
```

Comment:
Updating mocha was the important bit, all of the libraries indicated in #15 were included as dependencies to mocha.